app/main/views.py

```diff
@@ -33,7 +33,10 @@ def index():
     if req_opt_desc:
         for val in req_opt_desc.values:
             req_options[val] = False
-    return render_template('main/index.html', options=options_dict, req_options=req_options, req_desc=req_opt_desc)
+    twilio_auth = False
+    if os.environ.get('TWILIO_AUTH_TOKEN') is not None:
```

Comment:
Hi! Things have been changed -- instead of just checking os.environment, you also have to check TwilioAuth in SiteAttributes! Sry!!